packages/manager-tools/manager-generator/template/src/App.constants.ts

```diff
@@ -0,0 +1,148 @@
+
+/**
+ * Application identifier (injected by generator).
+ *
+ * @remarks
+ * Used across routing, tracking, and display.
+ */
+export const appName = '{{appNameKebab}}';
+
+/**
+ * Static application-level configuration.
+ *
+ * @remarks
+ * Includes labels and UI defaults, independent of feature flags.
+ */
+export const AppConfig = {
+  listing: {
+    datagrid: {
+      /** Service key used by Datagrid (injected at generation). */
+      serviceKey: '{{serviceKey}}',
+    },
+  },
+  /** Display label for the root navigation context. */
+  rootLabel: appName,
+} as const;
+
+/**
+ * Default API data mode for this app.
+ *
+ * - `'live'`: Network calls to real API endpoints.
+ * - `'mock'`: Local placeholder data from constants (offline/dev mode).
+ */
+export const API_DATA_MODE: 'live' | 'mock' = 'live';
+
+const docUrl = 'https://docs.ovh.com';
+
+/**
+ * Default onboarding configuration for mock or fallback mode.
+ *
+ * @remarks
+ * Provides product identity, brand, category, hero image (optional),
+ * onboarding tiles, and guide links.
+ */
+export const ONBOARDING_CONFIG: OnboardingConfigType = {
+  productName: '{{appNameKebab}}',
+  productCategory: 'Public Cloud',
+  brand: 'OVHcloud',
+  // Example hero:
+  // heroImage: { src: '/assets/assets/hero.png' },
+  tiles: [
+    { id: 1, key: 'guide1', linkKey: 'discover' },
+    { id: 2, key: 'guide2', linkKey: 'tutorial' },
+    { id: 3, key: 'guide3', linkKey: 'faq' },
+  ],
+  links: {
+    discover: docUrl,
+    tutorial: docUrl,
+    faq: docUrl,
+  },
+};
+
+/** API family selection types for generated code. */
+export type ListingApi = 'v6Iceberg' | 'v6' | 'v2';
+export type OnboardingApi = 'v6' | 'v2';
+
+/**
+ * API routes used by onboarding and listing fetchers.
+ *
+ * @remarks
+ * - `ONBOARDING_API_ROUTE`: Resolved with `{domain}` param.
+ * - `LISTING_API_ROUTE`: Root for listing requests.
+ */
+export const ONBOARDING_API_ROUTE = '{{onboardingEndpoint}}';
+export const LISTING_API_ROUTE = '{{listingEndpoint}}';
+
+/**
+ * Feature flags & routing configuration for this app.
+ *
+ * @remarks
+ * Drives the runtime behavior of routing, API selection, and tracking.
+ *
+ * Keys:
+ * - `isPci`: Toggles PCI flavor behaviors.
+ * - `routeFlavor`: Determines how root path is computed:
+ *   - `'pci'`           → `/[basePrefix]/pci/projects/:projectId/<appSlug>`
+ *   - `'generic'`       → `/[basePrefix]/<appSlug>`
+ *   - `'platformParam'` → `/[basePrefix]/:platformId`
+ * - `basePrefix`: Optional prefix before the flavor path (defaults to auto-detect).
+ * - `serviceParam` / `platformParam`: Route parameters (without `:`; normalized at runtime).
+ * - `appSlug`: Slug segment after the flavor root. For PCI, must be the short slug.
+ * - `tracking`: Configuration for analytics (region → level2, universe, subUniverse).
+ */
+export const APP_FEATURES = {
+  /** Listing API family to use (v6 Iceberg, v6, or v2). */
+  listingApi: '{{listingApi}}' as ListingApi,
```

Comment:
Listing API endpoint normally becomes an input for data-fetching hooks. So storing it as constant only for listing page deviates from the rest of the development.